### PR TITLE
Use sysctl Ansible module

### PR DESCRIPTION
**Pull request by electrocucaracha**
This change implements this [change](https://github.com/nephio-project/sig-release/issues/37) and fixes the Vagrantfile and gitea default user defined in the Bootstrap Ansible role

### CODE REVIEW

File: e2e/provision/playbooks/roles/bootstrap/defaults/main.yml [hunk 1]
```diff
@@ -18,5 +18,5 @@ kubernetes_version: v1.27.1
 gitea_postgres_password: c2VjcmV0  # echo -n "secret" | base64
 gitea_db_password: c2VjcmV0
 
-gitea_username: admin
+gitea_username: nephio
```
Comment (johnbelamaric) [inline e2e/provision/playbooks/roles/bootstrap/defaults/main.yml:21]: Do we have to change anything anywhere else because of this (e.g., in the gitea package?)

**Review by johnbelamaric — COMMENTED** [review]
on question but otherwise lgtm

/approve

File: e2e/provision/playbooks/roles/bootstrap/defaults/main.yml (hunk 1 above)
Comment (electrocucaracha) [reply e2e/provision/playbooks/roles/bootstrap/defaults/main.yml:21]: No, this is only taken when the integration tests are executed for the bootstrap role ([molecule test](https://github.com/nephio-project/test-infra/blob/main/e2e/provision/playbooks/roles/bootstrap/molecule/default/converge.yml#L16)), in the e2e is using [nephio](https://github.com/nephio-project/test-infra/blob/main/e2e/provision/nephio.yaml#L14)

**Comment by electrocucaracha** [comment]
> on question but otherwise lgtm
> 
> /approve

I just noticed that I need to [fix the terraform scripts](http://prow.nephio.io/view/gs/prow-nephio-sig-release/pr-logs/pull/nephio-project_test-infra/60/e2e/1664700379373244416#1:build-log.txt%3A222). I'll include the fix here as well

**Review by vishwanathj — APPROVED** [review]
/lgtm

**Comment by electrocucaracha** [comment]
/retest

I'm not sure why this took more than 2hrs to complete

File: e2e/provision/Vagrantfile [hunk 2]
```diff
@@ -20,6 +20,9 @@ Vagrant.configure('2') do |config|
   config.vm.box_check_update = false
   config.vm.synced_folder './', '/vagrant'
 
+  config.vm.network 'forwarded_port', guest: 7007, guest_ip: '127.0.0.1', host: 7007
```
Comment (johnbelamaric) [inline e2e/provision/Vagrantfile:23]: nice, so in the vagrant case you do not need the -L in your ssh? We should document that. You will still need to do the kubectl port forwarding, unless we expose the UI on a 172.18.0.0/20 address. Which I think in general would be fine for the sandbox, even though there is no authentication on the UI in sandbox mode. I'll create a separate issue to discuss that.

File: e2e/provision/Vagrantfile [hunk 3]
```diff
@@ -20,6 +20,9 @@ Vagrant.configure('2') do |config|
...
+  config.vm.network 'forwarded_port', guest: 3000, guest_ip: '172.18.0.200', host: 3000
```
Comment (vjayaramrh) [inline e2e/provision/Vagrantfile:24]: In generatl, might be a good idea to open a separate PR when adding other functionalities.
